src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+
+body {
+  --large-text-color: #060b35;
+  --small-text-color: #616070;
+  --button-color: #00acdc;
+}
+
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid white;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__img img {
+  height: 134px;
+  width: 160px;
+}
+
+.card__description {
+  padding-top: 40px;
+}
+
+.card__name {
+  color: var(--large-text-color);
+  padding-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__code {
+  color: var(--small-text-color);
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card__reviews {
+  padding-top: 16px;
+  padding-bottom: 24px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  color: var(--large-text-color);
+}
+
+.card__price {
+  padding-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.price {
+  color: var(--small-text-color);
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.sum {
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--small-text-color);
+  font-weight: 700;
+  text-shadow: none;
+}
+
+.card__link {
+  font-size: 14px;
+  line-height: 16px;
+  color: white;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  background-color: var(--button-color);
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  border: 1px solid var(--button-color);
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.card__link:hover {
+  background-color: white;
+  color: var(--button-color);
+}
```

Comment:
To make changes smooth on hover, add a 'transition' property to '.card__link', for example: 'transition: background-color 0.3s, color 0.3s'.